care/facility/migrations/0103_auto_20200425_1440.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.11 on 2020-04-25 09:10
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Similar lines in 2 files](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5419355)